spec/requests/linking/link_defendant_maat_reference_spec.rb

```diff
@@ -117,7 +118,7 @@
       it 'sends token request' do
         expect(
           a_request(:post, %r{.*/oauth/token})
-        ).to have_been_made.once
+        ).to have_been_made.twice
```

Comment:
?! this implies an issue exists

Reply:
@jsugarman is this not because we are now calling load_and_authorize_search to set the defendant in laa_refernces_controller as well as link_laa_reference (so two calls to the adaptor).

Reply:
as per discussion